notion-server/src/ethereum/dispatch.rs

```diff
@@ -14,7 +14,7 @@ enum Operation {
         Sender<Result<args::ExecutionEnvironment>>,
     ),
     GetShardBlock(args::GetShardBlock, Sender<Result<args::ShardBlock>>),
-    GetSimulationState(args::GetSimulationState, Sender<args::SimulationState>),
```

Comment:
Small note: It's likely that in the future we may need/want to pass some arguments to the simulation state function, but I'm ok with taking this out for now until that time comes.